pkg/new-ui/v1beta1/frontend/src/app/constants/algorithms-settings.const.ts

```diff
@@ -78,6 +78,8 @@ export const TPESettings: AlgorithmSetting[] = [
   },
 ];
 
+export const MultivariateTPESettings: AlgorithmSetting[] = [];
```

Comment:
Multivariate TPE accepts the following algorithm settings:

* `startup_trials` : integer
* `ei_candidates` : integer
* `random_state` : integer

https://github.com/kubeflow/katib/blob/7439a3762fd94ca700264302bd885f4565b57c11/pkg/suggestion/v1beta1/optuna/service.py#L72-L77